spec/acceptance/exec_powershell_spec.rb

```diff
@@ -34,6 +34,31 @@
 
   end
 
+  describe 'should handle a try/catch successfully' do
```

Comment:
FWIW, I had rewritten the test a bit differently under vanilla Beaker because I don't think that this test actually demonstrates the `catch` block is being executed, which I think we probably wanted to verify.

You can see what I did at https://github.com/puppetlabs/puppetlabs-powershell/pull/88/files#diff-655118f03be5e9bd030d84145ab98dd5R1

This test also is a bit subtle in that it assumes that `Write-Error` inside the `catch`, if it were run, would propagate something back to Puppet (which is a separate thing to verify IMHO).  I think the rewrite is a bit more explicit as it uses the scripts exit code in an `onlyif` to determine whether or not to write a file in the `command` portion.